db/migrate/20230718021133_add_seven_point_rating_to_surfline_v2s.rb

```diff
@@ -0,0 +1,6 @@
+class AddSevenPointRatingToSurflineV2s < ActiveRecord::Migration[7.0]
+  def change
+    add_column :surfline_v2_lolas, :seven_point_rating, :decimal
+    add_column :surfline_v2_lotus, :seven_point_rating, :decimal
```

Comment:
Any reason you made these `decimal` instead of `integer`? I believe they're always ints, like the wind/swell rating.

Reply:
There have been a couple of instances where the value is a decimal. Here's an example: 


<img width="323" alt="Screenshot 2023-08-07 at 7 24 43 AM" src="https://github.com/swrobel/meta-surf-forecast/assets/112027653/1a98568d-c20a-4ddb-b3db-fd697ecd85bb">